packages/terra-alert/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Unreleased
 
 * Added
-  * Added focus shift to the rendered notification banners content when notification is an alert and includes an action element
+  * Added focus shift to rendered `alert` notification banners content that include an interactable action element and/or dismiss.
 
 * Changed
   * Locked `uuid` to `8.2.0` to maintain IE compatibility.
```

Comment:
This change was already released so can we rebase with main so this PR is up to date?

Reply:
Rebased changes with main and updated changelog here https://github.com/cerner/terra-core/pull/3823/files#diff-b62ab7ae03f56c47f8aba033b108058b187c6bb44ff7dbe93a72c454c1393279